gen/generator.jl

```diff
@@ -19,9 +19,10 @@ options = load_options(joinpath(@__DIR__, "generator.toml"))
 # append!(options["general"]["output_ignorelist"], extract_inlined_funcs(headers))
 
 # Add compiler flags
-args = get_default_args()
+args = get_default_args("x86_64-w64-mingw32")
 push!(args, "-Iheaders")
 push!(args, "-std=c99")
+push!(args, "-DUA_ARCHITECTURE_POSIX")
```

Comment:
this is just for including C99 headers. not sure how to correctly setup compiler arguments for Windows.